src/components/RecipeCardStack.tsx

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import { StyleSheet, Dimensions, View, Text } from "react-native";
+import { useSelector, useDispatch } from "react-redux";
+import { addtoUserRecipeList } from "../redux/actions"
+import axios from "axios";
+import { colorPalette, shadowStyle } from "../constants/ColorPalette";
+import CardStack, { Card } from "react-native-card-stack-swiper";
+import RecipeCard from "../components/RecipeCard";
+import SwipeButtons from "../components/SwipeButtons";
+
+// Importing JSON data for development and testing
+import * as recipesJson from "../data/recipes.json";
+import { initialState } from "../redux/reducers/recipe"
+import { Recipe, RootState, UserState, FiltersState, UserRecipeListState, LoggedInParamList, recipeCardStackParamList } from "../../types";
+
+const _screen = Dimensions.get("screen");
+
+export default function RecipeCardStack({ randRecipes, filtersState }: recipeCardStackParamList) {
```

Comment:
I wonder if, by some miracle, there is any chance accessing the FilterState global variable with ```React.useSelector()``` might work from here.